### ISSUE TITLE
bf luis:cross-train creates invalid config internally

### ISSUE THREAD

**Issue by tomlm** [microsoft/botframework-cli#998]
## Versions
4.10.1

## Describe the bug
The code which generates a config by inspecting the file system doesn't generate the same data structure that the library that consumes is expecting.
in

in confighelp.js line 87
```javascript
          result[key].triggers[trigger.intent] = relativePath
```

in crosstrain.js line 38
```javascript
const {rootIds, triggerRules, intentName, verbose} = crossTrainConfig
```

You will see that the confighelper is creating a property called "triggers" while the library is expecting "triggerRules"

> THIS LIBRARY SHOULD HAVE BEEN TYPESCRIPT, this bug could have happened if the config had an interface definition.  There is a reason typescript exists and THIS IS IT.

[bug]

**Comment by feich-ms** [comment]
I have a PR here https://github.com/microsoft/botframework-cli/pull/990 to refactor the interface of crosstrain core lib. After the optimizations, the config will be aligned between composer and cli.

**Comment by feich-ms** [comment]
Closed by #1024, there is no config discovery and --config is required in cli now.
